doc/release/release.howto.txt

```diff
@@ -14,6 +14,841 @@
+
+See target/VOTE.txt. This must be heavily edited. It is useful to generate the release hashes.
+
+Call to vote by sending a message to the "dev" ML with subject
+"[VOTE] Release Commons Numbers 1.0 based on RC1". You can use the following example as
+a reference point, replacing the URLs with the appropriate ones:
+----------
+We have fixed quite a few bugs and added some significant enhancements since Apache Commons Numbers 1.0-beta1 was released,
+so I would like to release Apache Commons Numbers 1.0.
+
+Apache Commons Numbers (full distribution) 1.0 RC1 is available for review here:
+    https://dist.apache.org/repos/dist/dev/commons/numbers/1.0-RC1 (svn revision 48777)
+
+The Git tag commons-numbers-1.0-rc1 commit for this RC is 89a9c3817222a6c67d7a231b19f6c5c7fc995208 which you can browse here:
+    https://gitbox.apache.org/repos/asf?p=commons-numbers.git;a=commit;h=89a9c3817222a6c67d7a231b19f6c5c7fc995208
+You may checkout this tag using:
+    git clone https://gitbox.apache.org/repos/asf/commons-numbers.git --branch commons-numbers-1.0-rc1 commons-numbers-1.0-rc1
+
+Maven artifacts are here:
+    https://repository.apache.org/content/repositories/orgapachecommons-1556/
+
+These are the artifacts and their hashes:
+
+commons-numbers-1.0-bin.tar.gz=5e61ee22980a7534fb87793bb16eac371d7aea703552f2c8f14f049ae9646f97db8ae7e0e021c6ef4b4f1c78b529e33995fd4893030280437baf3081d429957a
+commons-numbers-1.0-bin.zip=4805949e5fab3c016d35be692b79a587b60e35a5809cc5be0817bce224a9462a5891400085040803183c54f2ff71ebdb385bf5119c5148b801333168aec14444
+commons-numbers-1.0-src.tar.gz=08656e681624e2bf2434714f6060e6b02de673089cc3711a01dfc3f0640791feb689206f68dc323a7bd14cb7760770567edc36bcdbea38b7eb31407521de6316
+commons-numbers-1.0-src.zip=5e80cfe64207c5ce990e2bee3e6a7ab7d22260f9e8f28b936197276c02bcba0e771b23f3f06bdcb4984adee69b748c4ebd3cddea40530b9cf8def64b6d4c91f7
+
+(no need for .asc hashes!)
+I have tested this with:
+
+'mvn clean install site' using:
+
+***
+<mvn -version>
+***
+
+Details of changes since 1.0-beta1 are in the release notes:
+    https://dist.apache.org/repos/dist/dev/commons/numbers/1.0-RC1/RELEASE-NOTES.txt
+
+Site:
+    https://home.apache.org/~mattjuntunen/commons-numbers-1.0-RC1-site/
+    (note some *relative* links are broken and the 1.0 directories are not yet created - these will be OK once the site is deployed.)
+
+CLIRR Report:
+    N/A
+
+JApiCmp Report:
+    N/A
+
+RAT Report:
+    https://home.apache.org/~mattjuntunen/commons-numbers-1.0-RC1-site/rat-report.html
+
+KEYS:
+  https://www.apache.org/dist/commons/KEYS
+
+Please review the release candidate and vote.
+This vote will close no sooner that 72 hours from now.
+
+  [ ] +1 Release these artifacts
+  [ ] +0 OK, but...
+  [ ] -0 OK, but really should fix...
+  [ ] -1 I oppose this release because...
+
+Thank you,
+
+Matt Juntunen,
+Release Manager (using key 7DD53AEFEDF1C3D392B51EBE346F4FCECFB70B1A)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Secret Access Key ID. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)